### ISSUE TITLE
Review sound design (a11y team)

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#222]
The sound design is now fully functional for this sim.  @emily-phet requested that we have members of the a11y team take it for a test drive and give us any feedback before sharing it with the larger design team.  I've published a dev version at https://phet-dev.colorado.edu/html/greenhouse-effect/1.1.0-dev.16/phet/greenhouse-effect_en_phet.html for this purpose.  

Please try out the sound and let us know if you have an major concerns or suggestions on how to improve it.  Note that there are "Extra Sounds" present in this sim.  Please check off your name from the list when you're done looking at and listening to it.  We're thinking that ten to fifteen minutes of playing around with it should be sufficient.  Thanks!

- [x] @BLFiedler 
- [x] @terracoda 
- [x] @jessegreenberg 
- [x] @Matthew-Moore240

**Comment by Ashton-Morris** [comment]
- On Waves Screen: Not sure if its intended but if you have Show Surface Temp Checked but Show Thermometer Unchecked you can still hear the Temperature ambiance. 

- On Layer Model Screen: You hear temperature sounds regardless of if all thermometers are unchecked.

**Comment by jessegreenberg** [comment]
I played with the sim for a while and I think I heard all of the sounds and understood what they are representing. I just noticed one bug: 

- Switching screens while the Flux Meter's "Extra Sounds" are playing causes those sounds to be stuck playing forever on the new screen. Going back to the original screen with the flux meter allows the sounds to finish playing and stop.

**Comment by Matthew-Moore240** [comment]
I think the sounds are good! I didn't notice anything wrong other than the few things others have noted.

**Comment by jbphet** [comment]
@Ashton-Morris said:

> On Waves Screen: Not sure if its intended but if you have Show Surface Temp Checked but Show Thermometer Unchecked you can still hear the Temperature ambiance.

This is intentional, and I recall it being specifically requested, though I'm not sure by whom.

Also,

> On Layer Model Screen: You hear temperature sounds regardless of if all thermometers are unchecked.

I should probably hook this sound to the ground temperature for consistency with the other screens.

- [ ] Disable sound on Layer Model screen when ground thermometer is off.

@jessegreenberg noticed that switching screens while the flux meter was producing sounds caused the sound to play forever.  Nice catch.

- [ ] Make flux meter sounds turn off when flux meter is not visible, such as when switching screens

In response to @BLFiedler's comment about not being able to pick out the flux meter sound, @Ashton-Morris and I are thinking that this might be an opportunity to try out the ducking feature that is going to be implemented soon.  Also, we will see if this comes up in interviews.

- [ ] Log an issue to try out ducking in conjunction with the flux meter and ambient sounds.

In response to @BLFiedler's comment about his daughter not liking the sound when the GHGs were high, we will see if we get any similar comments from others during interviews.

**Comment by jbphet** [comment]
In response to @terracoda's comment:

> When changing time periods I don't heard a lot of change in the surrounding sound. I think that might be intentional as GH gas levels (on the huge scale of things) are pretty close together. When comparing between zero GHG on the slider and then time periods, I made out differences better.

I'll bring the question of whether we want something more distinct to the larger design team.  We could potentially adjust the volume curve to make the sound of IR waves coming back more noticeable in this range.

- [ ] Discuss behavior of sound over Ice Age to 2020 range with design team, decide on whether to make any changes.

Another things that @terracoda said was:

> I think the sound tip sheet will be very helpful for this sim.

We should make sure this happens.

- [ ] Log an issue for making sure that tips about the sounds are created, either in the Teacher Tips doc or whether else they should go.

**Comment by jbphet** [comment]
On of the checkboxes above says:

> Discuss behavior of sound over Ice Age to 2020 range with design team, decide on whether to make any changes.

I've now discussed this with both the sound design team and the general design team, and the consensus is that we shouldn't change anything for this.  The difference in concentration is relatively subtle, though it can have significant impacts on a human scale, so we should represent it as such with the sound.

That means everything has been addressed for this issue.  Closing.
